go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	k8s.io/api v0.24.0
 	k8s.io/apimachinery v0.24.0
 	k8s.io/client-go v0.24.0
+	k8s.io/klog/v2 v2.60.1
```

Comment:
logrus is used everywhere else in this project, is there a reason to introduce klog here instead?

Reply:
No will switch